templates/participants/single_participant_home.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'web/app/app_base.html' %}
+{% load waffle_tags %}
+
+{% block breadcrumbs %}
+  {% load waffle_tags %}
```

Comment:
duplicate `load` tag (`waffle_tags` already loaded above). Also, I don't think it needs to be loaded at all.

Reply:
Ah yes, you are correct